crates/fsrt/src/main.rs

```diff
@@ -425,10 +346,28 @@ fn main() -> Result<()> {
         .with(EnvFilter::from_env("FORGE_LOG"))
         .init();
     let dirs = std::mem::take(&mut args.dirs);
-    let function = args.function.as_deref();
+
     for dir in dirs {
+        let mut manifest_file = dir.clone();
+        manifest_file.push("manifest.yaml");
+        if !manifest_file.exists() {
+            manifest_file.set_extension("yml");
+        }
+        debug!(?manifest_file);
+
+        let manifest_text = fs::read_to_string(&manifest_file)
+            .into_diagnostic()
```

Comment:
```suggestion
            .into_diagnostic()?;
```

We should be returning an error if there's no `manifest_file`, since we can't scan.

Reply:
Fixed. Thank you :)